packages/server/lib/controllers/connection.controller.ts

```diff
@@ -546,7 +551,7 @@ class ConnectionController {
                     type: template.auth_mode,
                     access_token,
                     refresh_token,
-                    expires_at,
+                    expires_at: expires_at || parsedExpiresAt,
```

Comment:
```suggestion
                    expires_at: parsedExpiresAt,
```
parseRawCredentials always return `expires_at` so I don't think you need the `||`
https://github.com/NangoHQ/nango/blob/41ee3c02a3b36b2b7a4cf55eaa9f094697f65a93/packages/shared/lib/services/connection.service.ts#L541-L542

Reply:
I think I did this because I didn't want to change the behavior of the import connection endpoint too much. If they provide an `expires_at` my original thinking was to leave it untouched if provided which is why I added the `||`.

But looking at it further maybe it is better that we do process the `expires_at` to run it through the `parseTokenExpirationDate` method to have a properly formatted date in which case your suggestion would make sense.

Thoughts?

Reply:
What happens right now if I pass an badly formatted expires_at to a `POST /connection/`?
If it accepts it, does that mean we have another bug that would be fixed by always calling `parseTokenExpirationDate`?

Reply:
Looks like it might be API specific. From a couple of quick tests:

1. Passing a random string "some-wack-date" -- in each scenario it lets it go through and keeps that string as expires_at
2. Passing in "400" in the case of Google gets accepted and the expires_at just defaults to the normal expiration time of one hour

FWIW this endpoint doesn't get called that much so probably too much time has already been invested into these potential cases